tensorflow/python/ops/signal/dct_ops.py

```diff
@@ -151,7 +149,8 @@ def dct(input, type=2, n=None, axis=-1, norm=None, name=None):  # pylint: disabl
 # TODO(rjryan): Implement `n` and `axis` parameters.
 @tf_export("signal.idct", v1=["signal.idct", "spectral.idct"])
 def idct(input, type=2, n=None, axis=-1, norm=None, name=None):  # pylint: disable=redefined-builtin
-  """Computes the 1D [Inverse Discrete Cosine Transform (DCT)][idct] of `input`.
+  """Computes the 1D [Inverse Discrete Cosine Transform (DCT)](https://en.wikipedia.org/wiki/Discrete_cosine_transform#Inverse_transforms)
+  of `input`.
```

Comment:
The short description shouldn't be longer than a single line (it'll confuse our linters). Can you change that to be the same style as you did in tensorflow/python/layers/normalization.py? That style is brief and still very nice. In this particular case (where the link is to wikipedia), I'd just leave the first line description as 
"Computes the 1D Inverse Discrete Cosine Transform (IDCT)."
and just add the reference at the bottom.

Reply:
While going through the docs, I chose to only use the `References:` section in the docs for references to scientific works and to keep the wikipedia and tensorflow links in-line. I just stumbled on this, because it is the only place where this reference style has been used and I figured it could be made consistent as well. 

Maybe the easiest/most consistent solution would be to revert my changes in this case?

Reply:
Yes, it may be fine to revert these cases.